test/jit/test_module_interface.py

```diff
@@ -405,3 +405,46 @@ def forward(self, input):
 
         scripted_mod.proxy_mod = NewScriptModule()
         self.assertEqual(scripted_mod(input), input * (input + 1) + 1)
+
+    # The call to forward of proxy_mod cannot be inlined. Making sure
+    # Freezing is perserving proxy_mod.
+    def test_freeze_module_with_interface(self):
```

Comment:
note from offline discussion: we are not always able to inline interface calls; we should throw for now if we can't

Reply:
Throwing an error as we discussed